test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_performance_optimized_sm_training.py

```diff
@@ -52,6 +53,7 @@ def test_optimized_tensorflow_sagemaker_training_performance_multinode(tensorflo
     '''
 
 
+@pytest.mark.skip("Skipped in favor of Training Compiler tests")
```

Comment:
Why are these skipped? For older versions without the training compiler, they will also be skipped. Are these tests redundant with the trcomp tests?

Reply:
Yes, they were a precursor to Training Compiler. We will not need to run these tests anymore.

Reply:
This will skip the test indefinitely, even for old versions of TF that don't have the training compiler. If this is expected behavior, why not just remove the functions? 

If it is not expected, and we need to run this test for older versions of TF still (that don't have trcomp), then this should be a conditional skip

Reply:
I am ok with removing these tests.